xbmc/GUIInfoManager.cpp

```diff
@@ -188,6 +188,18 @@ typedef struct
   int  val;
 } infomap;
 
+const infomap string_bools[] =   {{ "isempty",          STRING_IS_EMPTY },
+                                  { "equals",           STRING_EQUALS },
+                                  { "startswith",       STRING_STARTS_WITH },
+                                  { "endswith",         STRING_ENDS_WITH },
+                                  { "contains",         STRING_CONTAINS }};
+
+const infomap integer_bools[] =  {{ "equals",           INTEGER_EQUALS },
+                                  { "greater",          INTEGER_GREATER_THAN },
+                                  { "greaterorequals",  INTEGER_GREATER_OR_EQUALS },
+                                  { "less",             INTEGER_LESS_THAN },
+                                  { "lessorequals",     INTEGER_LESS_OR_EQUALS }};
```

Comment:
because we need to add `Is` in front of all conditions `IsEquals`, `IsGreater`, `IsLessOrEquals`